README.md

```diff
@@ -1,95 +1,26 @@
+2. [Open the jupyter notebook](https://github.com/jupyter/docker-stacks#quick-start) in your browser.
 
-## Infrastructure
+### Goal
 
-* Create a docker image/docker compose
-* Continuous Integration
-* Continuous Deployment
+Use a [recurrent neural network](https://towardsdatascience.com/illustrated-guide-to-recurrent-neural-networks-79e5eb8049c9) to produce new fortune cookie fortunes. This repo shows three different attempts to produce a working model to produce new fortunes.
 
-## App
+### Progress Thus Far
 
-* Render model results as plain text on an html page
-* Build flask prototype
-* Get a domain
-* Website-to-model 
-* Feedback from within UI
+* Attempt One: GRU with corpus embedding. Largely based on this blog post. TODO: Add blog post link.
+* Attempt Two: Pretrained embedding layer using [glove](https://nlp.stanford.edu/projects/glove/).
+* Attempt Three: Fine tune GPT-2 within a docker container locally.
 
+### Barriers
 
-## Marketing
-
-* Documentation of process/design/implementation
-* Write blog(s)
-  * decide where the blog belongs
-* Polish git repo
-
-
-## Things we've accomplished
-
-### September 5th
-* created a github repo
-* developed project plan
-* listed backlog items
-* renamed .ipynb notebook
-
-### September 12th
-* Created docker image and a docker-compose file
-
-### September 26th
-
-Things we talked about:
-• Adding a metric to the results of the output of the model.
-• We attempted to save our model as a static file for use later. We ran into issues in using different versions of keras.
-
-We cleaned some data, and did a git commit to show those changes in the repo. We retrained our model on our newly cleaned data.
-
-### Nov 14th
-
-Move ipython notebook to a .py file, cleaned up the script. 
-
-To-do: 
-- make a python package
-- add flask
-
-### Nov 7th & Oct 30th
-
-Added word embedding as an additional layer.
-
-### Oct 10th
-
-Things we've done:
-* Went through fortunie cookie ipython notebook
-* Figured out the tokenizer 
-  * tokenizer: strips out the punctuations
-  * padding: addes 0's to the beginning of the sentences
-
-To-do:
-- Update ipython notebook to the new data.csv
-- Some research to improve the model preformance
- - How to automatically stop a sentence
- - How to bring back the puncatuation (end stopping word) back to the sentence
- - How to fit the begin word smartly???
-- Update the kares version
-- Clean up the code and variable names 
-
-### Oct 17th
-
-Things we've done:
-- Tested the tensorflow character-level, turns out not work well.
-- Added word embedding (glove) to the fortune cookie model
-
-To-do:
-- finish the word embedding
-
+1. It has been difficult to find the volume of data required to train a natrual language processing model on fortune cookies. There are not very many fortune cookie databases out in the wild.
+2. Fortune cookies are built from somewhat atypical English, which has made the lack of data more difficult.
```

Comment:
I'd also maybe add to this point or one under it something about the difficulty that comes with testing. Challenges around verifying how "fortune cookie-y" our generated fortunes are.

Reply:
Agreed! I remembered we tried to score it manually, but still having the questions on finding an objective metric.